doc/extending/xlsx/xlsx_import.md

```diff
@@ -65,7 +65,8 @@ NW = no warning generated for these conversions.
     - fill/justify -> left
     - centerContinuous/distributed -> center
   - some vertical alignements. We only support top/center/bottom. Other types will be converted to center.
-  - other align options (wrapText, indent, shrinkToFit, ...) (NW)
+  - some wrappingText modes. We only support wrap/overflow imported from Google Sheets and Excel, while o-spreadsheet does have clip mode for text wrapping. Clip mode from Google Sheets will be converted to overflow mode.
```

Comment:
```suggestion
  - some wrappingText modes. We only support wrap/overflow imported from Google Sheets and Excel. Clip mode from Google Sheets will be converted to overflow mode, because even if o-spreadsheet support clip mode, it isn't supported by xlsx files.
```
a bit clearer IMO